clients/spark-extensions-base/src/main/scala/org/apache/spark/sql/execution/datasources/v2/NessieUtils.scala

```diff
@@ -238,20 +238,32 @@ object NessieUtils {
   }
 
   def getCurrentRef(
+      api: NessieApiV1,
```

Comment:
No need to create a new api instance, because all call sites have one available.